src/EA.Weee.Web/Areas/AatfReturn/Views/NonObligatedValuesCopyPaste/Index.cshtml

```diff
@@ -24,13 +24,13 @@
         </div>
     </header>
     <p class="govuk-body">
-        Provide tonnage to 3 decimal places for 14 categories. For example 1.238
+        Provide tonnage to 3 decimal places for @Html.DisplayFor(m => m.WeeeCategoryCount) categories. For example 1.238
     </p>
     <p class="govuk-body">
         You can copy the WEEE in tonnes from an Excel spreadsheet
     </p>
     <p class="govuk-body">
-        Data entered here will be interpreted as category 1 at the top through to category 14 at the bottom
+        Data entered here will be interpreted as category 1 at the top through to category @Html.DisplayFor(m => m.WeeeCategoryCount) at the bottom
```

Comment:
Please see https://github.com/DEFRA/prsd-weee/compare/150ba3cc87c330c346c9a3e3f4f327cfb1aea7fc..b7e5ce05c7fe9a6b5b6ba49a34456fe54c21985c -- this shouldn't be the count, but the max ID. It probably makes sense to use the same implementation as for the one that's already merged (and if there's an easy way to make it shared rather than copying, that'd be great too!)

Reply:
the reason for using separate properties when I implemented the original is it's simple to do, and otherwise you're introducing a place that does not support non-consecutive IDs.